modules/basegl/src/processors/instancerenderer.cpp

```diff
@@ -58,7 +58,27 @@ const ProcessorInfo InstanceRenderer::processorInfo_{
     "Mesh Rendering",               // Category
     CodeState::Stable,              // Code state
     Tags::GL,                       // Tags
-};
+    R"(
+        Renders multiple instances of a mesh.
+        Each instance can be modified using uniform data provided from a set of dynamic inports
+        holding vectors of data. The the number of inport and types can be controlled using
+        a List Property.
+        
+        The rendering will happen along these lines:
+        
+            for (auto uniforms : zip(dynamic vector data ports)) {
```

Comment:
Does this render as expected?
Might want to consider wrapping code in 
```
```cpp
<code>
<tripplequote ```>
```

Reply:
it does